src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -216,6 +217,12 @@ def cli_cosmosdb_keys(client, resource_group_name, account_name, key_type=Cosmos
     raise CLIError("az cosmosdb keys list: '{0}' is not a valid value for '--type'. See 'az cosmosdb keys list --help'.".format(key_type))
 
 
+def _handle_exists_exception(cloud_error):
+    if cloud_error.status_code == 404:
+        return False
```

Comment:
Is there any other error conditions when `status_code` is 404 other than where the resource already exists? 
And the original meaning of `404` as http staus should be **not found**, `cloud_error.status_code` is on the contrary, it seems misleading. If the error status provided by the server can be modified, I think it might be better to use `409 Conflict` for this scenario.

Reply:
It should be fine to use `404` for not found scenario. When not found, it means the db doesn't exist and return false for the command.